ios/PhotoEditorModule.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  PhotoEditorModule.swift
+//  Runner
+//
+//  Created by Banuba on 9.02.24.
+//
+
+import BanubaPhotoEditorSDK
+
+class PhotoEditorModule: BanubaPhotoEditorDelegate {
+    var photoEditorSDK: BanubaPhotoEditor?
+    
+    private var flutterResult: FlutterResult?
+    
+    init(token: String, flutterResult: @escaping FlutterResult) {
+        let configuration = PhotoEditorConfig()
+        photoEditorSDK = BanubaPhotoEditor(
+            token: token,
+            configuration: configuration
+        )
+        if photoEditorSDK == nil {
+            flutterResult(FlutterError(code: AppDelegate.errEditorNotInitialized, message: "", details: nil))
+            return
+        }
+
+        photoEditorSDK?.delegate = self
+
+        flutterResult(nil)
+    }
+    
+    func presentPhotoEditor(
+        fromViewController controller: FlutterViewController,
+        imagePath: String,
+        flutterResult: @escaping FlutterResult
+    ) {
+        self.flutterResult = flutterResult
+        
+        guard let image = UIImage(contentsOfFile: imagePath) else {
+                print("❌ Failed to load image from path: \(imagePath)")
+                flutterResult(FlutterError(code: "INVALID_IMAGE_PATH", message: "Image could not be loaded from the provided path.", details: nil))
+                return
+            }
+        
+        let launchConfig = PhotoEditorLaunchConfig(
+            hostController: controller,
+            entryPoint: .editorWithImage(image)
+        )
+        
+        photoEditorSDK?.delegate = self
+        
+        photoEditorSDK?.getLicenseState(completion: { [weak self] isValid in
+          guard let self else { return }
+          if isValid {
+            print("✅ License is active, all good")
+              photoEditorSDK?.presentPhotoEditor(
+                  withLaunchConfiguration: launchConfig,
+                  completion: nil
+              )
+          } else {
+            print("❌ License is either revoked or expired")
+          }
+        })
+    }
+    
+    // MARK: - PhotoEditorSDKDelegate
+    func photoEditorDidCancel(_ photoEditor: BanubaPhotoEditor) {
```

Comment:
Should this method be called in your`BanubaService`?

Reply:
no there is no any callback for it on documentation.